closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -666,6 +888,10 @@ JNIEXPORT jlong JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_createRSAPublic
 
     unsigned char* nNative;
     unsigned char* eNative;
+    RSA* publicRSAKey;
```

Comment:
Ideally these (and also in the subsequent changes to this file) should be initialized when declared (to NULL in this case). Which helps avoid future bugs.